src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -175,10 +181,11 @@ const Stream = React.createClass({
         logAjaxError(error);
         this.setState({
           loading: false,
+          isDefaultSearch: null,
```

Comment:
At this point, the component doesn't know whether a search is the default search or not ... I'm using `null` to differentiate between `false`. It could also be `undefined`.